deployment/Dockerfile

```diff
@@ -13,12 +13,13 @@
 # limitations under the License.
 
 FROM alpine:3.12.0@sha256:90baa0922fe90624b05cb5766fa5da4e337921656c2f8e2b13bd3c052a0baac1
+
 VOLUME /tmp
 
 ARG K8_VERSION=v1.15.3
 
-RUN apk --no-cache add curl=7.76.1-r0
-RUN apk --no-cache add bash=5.0.17-r0
+RUN apk --no-cache add curl
+RUN apk --no-cache add bash
```

Comment:
looking at the hadolint rule: https://github.com/hadolint/hadolint/wiki/DL3018 
We can specify a partial version using `=~5.0.17`
Update the curl and bash versions in the dockerfile to use the partial versions and update the hadolint yaml rules appropriatley

Reply:
Interesting about the partial version, I did not know about that.  I re-visited the ticket and it was specific to ask that we use the latest version of curl and bash.   We were even asked to add a hadolint.yaml to suppress any complaints.    I took all of this to mean that in this instance we don't care about the version.  As I'm not sure which way this should go, I have left it as is.